RubberduckTests/Refactoring/ReorderParametersTests.cs

```diff
@@ -140,5 +742,53 @@ private static Mock<IRefactoringPresenterFactory<IReorderParametersPresenter>> S
             factory.Setup(f => f.Create()).Returns(presenter.Object);
             return factory;
         }
+
+        private void SetupProject(string inputCode)
```

Comment:
Same deal. Either a change needed to be made to the base class, or a new subclass should be created.